docs/NMOS With H.264.md

```diff
@@ -58,11 +59,116 @@ The term 'strict-CBR' corresponds to the definition of constant bit-rate at sect
+  The Flow resource MUST indicate the H.264 profile, which defines algorithmic features and limits that MUST be supported by all decoders conforming to that profile. The stream's active parameter sets MUST be compliant with the `profile` attribute of the Flow. The permitted `profile` values are strings, defined as per ITU-T  Rec. H.264 Annex A
+
+  - "BaselineConstrained"
+  - "Baseline" (Default if not specified in the SDP transport file)
+  - "Main"
+  - "Extended"
+  - "High"
+  - "HighProgressive"
+  - "HighConstrained"
+  - "High10"
+  - "High10Progressive"
+  - "High-422"
+  - "HighPredictive-444"
+  - "High10Intra"
+  - "HighIntra-422"
+  - "HighIntra-444"
+  - "CAVLCIntra-444"
+
+Informative note: The names of the profiles in string form have been derived from the names used at Annex A of the H.264 standard with whitespace omitted, the sampling mode always positioned at the end of the string, preceded by a '-' and the terms High and Baseline positioned at the beginning of the string.
+
+The profile strings in this specification are included in the [NMOS Parameter Registers][]. Additional strings may be added there in the future.
+
+- [Level](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#level)
+  The Flow resource MUST indicate the H.264 level, which defines a set of limits on the values that may be taken by the syntax elements of an H.264 bitstream. The stream's active parameter sets MUST be compliant with the `level` attribute of the Flow. The permitted `level` values are strings, defined as per ITU-T Rec. H.264 Annex A
+
+  - "1" (Default if not specified in the SDP transport file)
+  - "1b", "1.1", "1.2", "1.3"
+  - "2", "2.1", "2.2"
+  - "3", "3.1", "3.2"
+  - "4", "4.1", "4.2"
+  - "5", "5.1", "5.2"
+  - "6", "6.1", "6.2"
+
+Informative note: The names of the levels in string form have been derived from the names used at Annex A of the H.264 standard.
+
+The level strings in this specification are included in the [NMOS Parameter Registers][]. Additional strings may be added there in the future.
+
+The Flow's `profile` and `level` attributes map to the `profile-level-id` parameter of the SDP transport file. See section SDP format-specific parameters.
+
+The Flow's `profile` and `level` attributes map to the members profile_idc, level_idc and constraint_set\<N\>_flag of the AVC_video_descriptor of an MPEG2-TS transport stream. See section Multiplexed Flows.
+
+Informative note: The Flow's `profile` and `level` attributes are always required. The SDP transport file `profile-level-id` parameter MAY be omitted when matching the default value.
+
+- [Bit Rate](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#bit-rate)
+  The Flow resource MUST indicate the target encoding bit rate (kilobits/second) of the H.264 bitstream. It MUST comply with the stream's active parameter sets. The `bit_rate` integer value is expressed in units of 1000 bits per second, rounding up.
```

Comment:
A similar confusing (for me) "comply with" here...

Reply:
done